app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -95,7 +95,7 @@ def export
         fail HttpErrors::BadRequest, _("Invalid usage for Pulp 3 repositories. "\
                                        "Use hammer content-export for Yum repositories")
       end
-      ::Foreman::Deprecation.api_deprecation_warning("Export is being deprecated and will be removed in a future version of Katello. Use hammer content-view version export instead.")
+      ::Foreman::Deprecation.api_deprecation_warning("Export is being deprecated and will be removed in Katello 4.1. Use hammer content-view version export instead.")
```

Comment:
@parthaa tagged you for this review to get your feedback here.  "Export is being deprecated" sounds scary, but I think this just refers to the pulp2 endpoint?  Is that correct? Is there a better way to say this?

Reply:
This is correct, you might change it to suggest the new endpoint?  And similar to the puppet parts, even though the api is there, its useless in 4.0.  Its basically non-functional

Reply:
Ok, changing it to
```
Pulp 2 export is deprecated and will not work as of Katello 4.0.  To export using Pulp 3, use hammer content-export version.
```